test/test_common/environment.h

```diff
@@ -93,6 +93,18 @@ class TestEnvironment {
    */
   static const std::string& nullDevicePath();
 
+  /**
+   * Obtain platform specific new line character(s)
+   * @return absl::string_view platform specific new line character(s)
+   */
+  static constexpr absl::string_view newLine() {
+#ifdef WIN32
+    return {"\r\n", 2};
+#else
+    return {"\n", 2};
```

Comment:
typo

Reply:
you don't need the second parameter, just {"\r\n"} and {"\n"} works.

Reply:
thanks @lizan fixed in 5d4541b08192db07bf8409c27b827dbff79f377b